content/en/docs/tutorials/github.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup GitHub Actions"
+subtitle: "Setup GitHub Actions for Continuous Integration"
+weight: 5
+draft: false
+description: >
+    A tutorial on how to set up Score with GitHub Actions.
+---
+
+You can use `score-humanitec` in your GitHub Action workflows, for example, when you make a code change, and you want the Humanitec Platform Orchestrator to deploy your code to a new environment.
+
+The following is a guide to setting up the Score implementation CLI in your GitHub Action workflows.
+
+For the Score GitHub Action, see [Setup Score](https://github.com/score-spec/setup-score)
+
+## Usage
+
+To use the Score GitHub Action, add the following step to your workflow:
+
+```yaml
+steps:
+  - uses: actions/setup-score@v2
+    with:
+      file: score-humanitec
+      version: '0.1.0'
+```
+
+This will download and cache the specified version of the `score-humanitec` CLI and add it to PATH.
+
+The action accepts the following inputs:
+
+- `file` - The Score CLI tool to install. For example, `score-humanitec`, `score-compose`, or `score-helm`.
```

Comment:
Following my 2 comments above: I think this should either be a `score-humanitec` tutorial or a generic tutorial using `score-compose` and `score-helm` as examples as we're aiming to move anything `score-humanitec` into the Humanitec docs eventually (not sure if I'm fully up to date here)